pkg/tests/tasks/traffic/circuit_breaking_test.go

```diff
@@ -69,7 +69,7 @@ func TestCircuitBreaking(t *testing.T) {
 			msg := oc.Exec(t,
 				pod.MatchingSelector("app=fortio", ns),
 				"fortio",
-				fmt.Sprintf("/usr/bin/fortio load -c %d -qps 0 -n %d -loglevel Warning http://httpbin:8000/get", connection, reqCount))
+				fmt.Sprintf("/usr/bin/fortio load -c %d -qps 0.1 -n %d -loglevel Warning http://httpbin:8000/get", connection, reqCount))
```

Comment:
qps 0.1 == 1 request per 10 seconds?

Reply:
I'm checking the documentation

Reply:
I just adjust the qps to 4, increase the execution time in 10 seconds but seems to be more stable in the balance of ko vs ok